### DISCUSSION TITLE
Deploying to phone device

### DISCUSSION THREAD

**Discussion by DAT4** [gokrazy/gokrazy#185] [Ideas]
Android is already build on a linux kernel right. It could be so nice to take a phone and install gokrazy on it - imagine the opportunities ! :)

**Comment by stapelberg** [comment]
I don’t mind if someone wants to work on that (possibly in a separate repository, though, depending on what’s needed for it), but personally I have no use-case for gokrazy on phones. 

These days, I use my phone as a phone. I don’t have a terminal emulator or SSH client installed on it, and I don’t plan to.

**Comment by jshield** [comment]
I have a number of android devices (mostly phones) that have varying levels of postmarketOS support, one use would be basically an SBC with a built in UPS, display, plus a set of other peripherals like GPS, etc. 

Main issue I can see is the desire to keep a reasonable level of mainline support, trying to wrangle all the various build trees is probably a major hassle, I wonder how hard it would be to build images that were fastload/TWRP adb sideload compatible

**Reply by DAT4** [reply]
I don't know much about how to make the image compatible. But I think if we could swap the kernel for the kernel used on android it would probably work.

Maybe the guys from postmarketOS, or the librem 5 team would know something.

I might be able to test something on a Pinephone.

Btw. What does SBC stand for? 

I was thinking to build a minimalist phone that uses Matrix or tox for comunicating, and I am really interested how much power this kind of phone would use.

**Reply by stapelberg** [reply]
> What does SBC stand for?

Single Board Computer, e.g. the Raspberry Pi.
